src/Umbraco.Web/WebApi/UnhandledExceptionLogger.cs

```diff
@@ -29,7 +29,10 @@ public override void Log(ExceptionLoggerContext context)
                 && context.ExceptionContext.ActionContext.ControllerContext.Controller != null
                 && context.Exception != null)
             {
-                _logger.Error(context.ExceptionContext.ActionContext.ControllerContext.Controller.GetType(), "Unhandled controller exception occurred", context.Exception);
+                var requestUrl = context.ExceptionContext.ControllerContext.Request.RequestUri.AbsoluteUri;
```

Comment:
Prob need to do some null checks on this (can use `?.`) since i think there might be cases where it's null